group_vars/all

```diff
@@ -4,6 +4,8 @@ app_package: flaskapp
 app_callable: app
 base_dir: '/srv/{{ app_name }}'
 nginx_server_name: 'localhost {{ app_name }}.local'
+nginx_http_port: 80
```

Comment:
Is this configuration necessary? `nginx_https_port` is used in a comment that needs to be manually handled, so it’s better to leave it hard-coded. What use-case do you have for a different HTTP port than 80 though? For development, the built-in server of your framework will be better. For real-life deployment, most people want the standard port 80.

(I will accept it if you convince me)